tools/lsp/common/rename_component.rs

```diff
@@ -160,24 +160,40 @@ fn fix_imports(
             exporter_path,
             old_type,
             new_type,
-            fixup_local_use,
+            local_rename_function,
             edits,
         );
     }
 }
 
+fn import_path(
+    document_directory: &Path,
+    import_specifier: &syntax_nodes::ImportSpecifier,
+) -> Option<PathBuf> {
+    let import = import_specifier
+        .child_token(SyntaxKind::StringLiteral)
+        .map(|t| t.text().trim_matches('"').to_string())?;
+
+    if import == "std-widgets.slint" || import == "std-widgets.s60" {
```

Comment:
```suggestion
    if import == "std-widgets.slint" {
```
It was called `sixtyfps_widgets.60` but we no longer support that

Reply:
I replaced this with

```rust
if import == "std-widgets.slint" || import.starts_with('@') {
```

We should not rename things imported from libraries in general.